package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-quick-chat",
-  "version": "1.2.8",
+  "version": "1.2.9",
```

Comment:
It's not needed to do it on this PR. I usually create a separate commit to change the package version.